config/features.yml

```diff
@@ -415,6 +415,10 @@ features:
       Toggle for the entire pre-entry covid 19 self-screener available at /covid19screener and to be used by visitors
       to VHA facilities in lieu of manual screening with a VHA employee.
       This toggle is owned by Patrick B. and the rest of the CTO Health Products team.
+   profile_always_show_direct_deposit_display:
+    actor_type: user
+    description: Shows the direct deposit link for all veterans
+    enable_in_development: true
```

Comment:
Syntax Error: Indentation is off.

Reply:
Updated the whitespace! Thanks for the catch!